api.md

```diff
@@ -412,6 +413,39 @@ queries.
 
 ---
 
+## CREATE INDEX [](create_index)
+
+```SQL
+CREATE INDEX ... WITH (timescaledb.transaction_per_chunk, ...);
+```
+
+This option extends [`CREATE INDEX`][postgres-createindex] with the
+ability to use a separate transaction for each chunk it creates an
+index on, instead of using a single transaction for the entire hypertable.
+This allows `INSERT`s, and other operations to to be performed concurrently
+during most of the duration of the `CREATE INDEX`.
+(While the the index is being created on an individual chunk, it functions as
```

Comment:
the the

Reply:
I don't think parentheses are needed here